react/week2/newtodo-list-app/src/Counter.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Counter = () => {
+  const [increment, changeIncrement] = React.useState(0);
```

Comment:
If you want you could use the destructured version here as you do in the ToDoListRow component:
import React, { useState} from "react";
```suggestion
  const [increment, changeIncrement] = useState(0);
```

Reply:
Thanks for the valuable comments. I will resolve all the suggestions. 

 Here I did not use useEffect for setting the timer. I could see in timer examples we have used useEffect. But without using that also the counter is working. Is it necessary to use useEffect here? please give me a suggestion. Thanks again for the great review.